src/import-export/azext_import_export/generated/_params.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from knack.arguments import CLIArgumentType
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_three_state_flag,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from azext_import_export.action import (
+    AddReturnAddress,
+    AddReturnShipping,
+    AddShippingInformation,
+    AddDeliveryPackage,
+    AddReturnPackage,
+    AddDriveList,
+    AddExport
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('import-export list') as c:
+        pass
+
+    with self.argument_context('import-export show') as c:
+        c.argument('name', options_list=['--name', '-n'], help='The name of the import/export job.')
+
+    with self.argument_context('import-export create') as c:
+        c.argument('name', options_list=['--name', '-n'], help='The name of the import/export job.')
+        c.argument('client_tenant_id', help='The tenant ID of the client making the request.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', tags_type)
+        c.argument('storage_account_id', help='The resource identifier of the storage account where data will be imported to or exported from.')
+        c.argument('type', help='The type of job')
+        c.argument('return_address', action=AddReturnAddress, nargs='+', help='Specifies the return address information for the job.')
+        c.argument('return_shipping', action=AddReturnShipping, nargs='+', help='Specifies the return carrier and customer\'s account with the carrier.')
+        c.argument('shipping_information', action=AddShippingInformation, nargs='+', help='Contains information about the Microsoft datacenter to which the drives should be shipped.')
+        c.argument('delivery_package', action=AddDeliveryPackage, nargs='+', help='Contains information about the package being shipped by the customer to the Microsoft data center.')
+        c.argument('return_package', action=AddReturnPackage, nargs='+', help='Contains information about the package being shipped by the customer to the Microsoft data center.')
+        c.argument('diagnostics_path', help='The virtual blob directory to which the copy logs and backups of drive manifest files (if enabled) will be stored.')
+        c.argument('log_level', help='Default value is Error. Indicates whether error logging or verbose logging will be enabled.')
```

Comment:
is this enum in swagger? if yes, pls feedback to code gen team.

Reply:
It's defined as `string` in swagger.